ConsoleCalculator/Calculator.cs

```diff
@@ -1,14 +1,127 @@
+                //toogle sign change for first Operand                             
+                if(keyPressed=="s" || keyPressed=="S")
+                    A=A*(-1);        
+                Console.Write($"{A}\n");
+            }
+               
+            //Extraction Of Operator==================================================================================================
+            else if(keyPressed=="+" || keyPressed=="-" || keyPressed=="x" || keyPressed=="X"|| keyPressed=="/")                             //extraction of operator
+            {
+                opertor=keyPressed;
+                if(Flag==true)              //for concussive operator continusly,flag tell presence of b
+                {
+                    try
+                    {
+                        Console.Clear();
+                        A=Calculate(A,B,opertor);
+                        test=A;                             //extra var to unit testing
+                        B=0;
+                    }
+                    catch(DivideByZeroException)
+                    {
+                        Console.Clear();
+                        Console.WriteLine("-E-");
+                        msg="error";
+                    }
+                }
+                if(msg=="")                             //do not print if exception occured
+                    Console.Write($"{A}\n{opertor}\n");  
+                Flag=true;
+                temp2="";   
+            }
+
+            //Extraction of 2nd Operand=========================================================================================
+            else if(keyPressed!="=" && Flag==true)               
+            {
+                if((keyPressed!="s" && keyPressed!="S") && (keyPressed!="c" && keyPressed!="C"))
+                {
+                    temp2=string.Concat(temp2,keyPressed);
+                    B=Double.Parse(temp2);
+                }
+                Console.Clear();
+                //toogle sign change for 2nd Operand
+                if(keyPressed=="s" || keyPressed=="S")                
+                    B=B*(-1);
+                Console.Write($"{A}\n{opertor}\n{B}\n");
+            }
+
+            else if(keyPressed=="=")                                //check for b is given or not
+            {
+                
+                temp=""; temp2="";
+                try
+                {
+                    Console.WriteLine($"{A}\n{opertor}\n{B}\n{keyPressed}\n{Calculate(A,B,opertor)}");
+                    A=Calculate(A,B,opertor);
+                    test=A;                                           //for unit testing
+                    opertor="";B=0;
+                }
+                catch(DivideByZeroException)
+                {
+                    Console.WriteLine("-E-");
+                }
+                    
+            }
+                
+        }
+
+        public static double Calculate(double A,double B,string opertor)
+        {

```

Comment:
Since Iam using using a Boolean variable which changes its value at runtime.So switch is not supporting and throwing me error as follows `Cannot implicitly convert type 'bool' to 'string' (CS0029) [ConsoleCalculator]`.So right now Iam not changing the whole logic of program, but I will always keep your advice from now onwards.
[https://www.whizlabs.com/blog/ocajp-switch/](I have taken reference from here)